crates/bitwarden-vault/src/cipher/local_data.rs

```diff
@@ -10,17 +10,17 @@ use tsify_next::Tsify;
 #[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
 #[cfg_attr(feature = "wasm", derive(Tsify), tsify(into_wasm_abi, from_wasm_abi))]
 pub struct LocalData {
-    last_used_date: Option<u32>,
-    last_launched: Option<u32>,
+    last_used_date: Option<u64>,
+    last_launched: Option<u64>,
```

Comment:
Hmm, u64 is BigInt in js world according to https://rustwasm.github.io/wasm-bindgen/reference/types/numbers.html. I wonder if we should just use Chrono date instead. @fedemkr do we use last_used_date in mobile?

Reply:
The one inside "Cipher -> Local data" is not being used as far as I see. Only in test mocking to comply with the initializer.
I've checked in Android as well but @SaintPatrck could you confirm just in case I've missed some spot?

Reply:
@gbubemismith I think we should switch to using DateTime<UTC> in that case. It's consistent with other usages.

Reply:
yeah that makes sense